app/Providers/AppServiceProvider.php

```diff
@@ -3,7 +3,7 @@
 namespace App\Providers;
```

Comment:
I would just create a variable projectId and let users replace that instead of replacing a string iside of the class init. Also, change this in the docs.

Reply:
a89ead5 - used .ENV variable for thst